lib/real_world/blog/blog.ex

```diff
@@ -23,7 +23,16 @@ defmodule RealWorld.Blog do
     limit = params["limit"] || @default_article_pagination_limit
     offset = params["offset"] || 0
     from(a in Article, limit: ^limit, offset: ^offset, order_by: a.created_at)
-    |> Repo.all
+      |> filter_by_tags(params["tag"])
```

Comment:
@rohanpujaris will this work if `params["tag"]` is not there ?

Reply:
@lbighetti If `params["tag"]` it returns the original ecto query without adding any clause.
 
Check [here](https://github.com/gothinkster/elixir-phoenix-realworld-example-app/pull/20/files#diff-19d821f3b6c494eb023139fcac49d57cR31)

Reply:
Alright sounds good!